drivers/hmis/app/models/hmis/form/assessment_processor.rb

```diff
@@ -30,7 +30,11 @@ def run!
 
       container, field = match[1..2]
 
-      container_processor(container)&.process(field, value)
+      begin
+        container_processor(container)&.process(field, value)
+      rescue StandardError => e
+        raise e.class, "Error processing field '#{field}': #{e.message}"
```

Comment:
This will result in more helpful errors for development. Specifically:
```
     RuntimeError:
       Error processing field 'physicalDisability': Unrecognized key 'NOT_REAL' for enum Types::HmisSchema::Enums::Hud::NoYesReasonsForMissingData
```